### ISSUE TITLE
VoiceOver (iOS) not reading ARIA state changes when what-input is loaded

### ISSUE THREAD

**Issue by davepenfold** [ten1seven/what-input#86]
I've created a test case here:

https://codesandbox.io/s/4wlrq70km0

### Test 1 

**Steps to reproduce:**

Open https://4wlrq70km0.codesandbox.io/ in Safari on your iOS device. Tested with version 11.4.1.

- double tap the "Click me" button
- VoiceOver should read button as "Click me, expanded"
- double tap the "Click me" button again

**Expected results:**

VoiceOver should read button as "Click me, collapsed"

**Actual results:**

VoiceOver does not announce the aria-expanded change

### Test 2 

- comment out Line no. 3 `import "what-input";` and click on Save button so what-input is not loaded
- repeat steps above, you will find you get the expected result this time.

**Comment by ten1seven** [comment]
Hi @davepenfold, thanks for reporting this. I checked out your example last night on my phone that's been upgraded to iOS 12 and found that double tapping the screen or directly on the button to initiate a click on the button produced what I would expect as a correct result of VoiceOver announcing "click me, expanded" and "click me, collapsed". A single click directly on the button did not activate it, but simply announced the button and its state.

Is it possible that the behavior that you're seeing is an issue with VoiceOver that was changed in iOS 12? Are you planning on upgrading in the near future so you can test on your end?
